recipe/bld_winapi.bat

```diff
@@ -0,0 +1,27 @@
+set LIB=%LIBRARY_LIB%;%LIB%
+set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
+set INCLUDE=%LIBRARY_INC%;%INCLUDE%;%RECIPE_DIR%
+
+:: Configure.
+:: -DZLIB_WINAPI switches to WINAPI calling convention. See Q7 in DLL_FAQ.txt.
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -D CMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% ^
+      -D CMAKE_C_FLAGS="-DZLIB_WINAPI " ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+:: Build.
+cmake --build %SRC_DIR% --config Release
+if errorlevel 1 exit 1
+
+:: Test.
+ctest
+if errorlevel 1 exit 1
+
+:: Copy built zlib.dll with the same name provided by http://www.winimage.com/zLibDll/
+:: This is needed for example for cuDNN
+:: https://docs.nvidia.com/deeplearning/cudnn/install-guide/index.html#install-zlib-windows
+copy "zlib.dll" "%LIBRARY_BIN%\zlibwapi.dll" || exit 1
+copy "zlib.lib" "%LIBRARY_LIB%\zlibwapi.lib" || exit 1
```

Comment:
A package like `libzlib` and `libzlib-wapi` should only have the dll. Maybe this file should go into another output `zlib-wapi` which would depend on both `zlib` and `libzlib-wapi`, so that a recipe needs to have `zlib-wapi` in host only.

Reply:
Thank you, Isuru! I didn't know about the .dll and .lib separation in packages. I've added `zlib-wapi`. Could you please take another look?

Reply:
Isn't `.lib` still in `libzlib-wapi`?

Reply:
Right, it's still there...
I need to figure out how I can reuse the build folder of `libzlib-wapi` to include the `.lib` file only in the `zlib-wapi` package.

Reply:
I moved building of wapi version to `bld.bat` simplifying the reuse of the build folder for `libzlib-wapi` and `zlib-wapi` packages.